mne/preprocessing/ica.py

```diff
@@ -2624,3 +2638,39 @@ def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
         return template_fig, labelled_ics
     else:
         return None
+
+
+def read_ica_eeglab(fname):
+    """Load ICA information saved in an EEGLAB .set file.
+
+    Parameters
+    ----------
+    fname : str
+        Complete path to a .set EEGLAB file that contains an ICA.
+
+    Returns
+    -------
+    ica : instance of ICA
+        An ICA object based on the information contained in the input file.
+    """
+    eeg = _check_load_mat(fname, None)
+    info, _, _ = _get_info(eeg)
```

Comment:
or

```suggestion
    info = _get_info(eeg)[0]
```

... which would make it robust against changing the number of return parameters and not lead to one of the `_`s lingering in memory.